.github/workflows/test.yml

```
@@ -172,12 +172,12 @@ jobs:
       - name: rust cache
         uses: Swatinem/rust-cache@v2
         with:
-          prefix-key: "v1-${{ runner.os }}-rust-cache"
+          prefix-key: 'v1-${{ runner.os }}-rust-cache'
```

might be a good time to simplify the e2e cache names
```suggestion
          prefix-key: 'v2-rust-main'
```

---

@paulbalaji good point, will do this in another PR just to avoid scope creep on an already very large PR

---

Updated ✅